Content.Shared/Maps/ContentTileDefinition.cs

```diff
@@ -14,17 +15,17 @@ public sealed class ContentTileDefinition : IPrototype, IIndexedPrototype, ITile
     {
         string IIndexedPrototype.ID => Name;
 
-        public string Name { get; private set; }
+        public string Name { get; private set; } = string.Empty;
         public ushort TileId { get; private set; }
-        public string DisplayName { get; private set; }
-        public string SpriteName { get; private set; }
+        public string DisplayName { get; private set; } = string.Empty;
+        public string SpriteName { get; private set; } = string.Empty;
         public bool IsSubFloor { get; private set; }
-        public List<string> BaseTurfs { get; private set; }
+        public List<string> BaseTurfs { get; private set; } = new();
         public bool CanCrowbar { get; private set; }
-        public string FootstepSounds { get; private set; }
+        public string FootstepSounds { get; private set; } = string.Empty;
         public float Friction { get; set; }
         public float ThermalConductivity { get; set; }
-        public string ItemDropPrototypeName { get; private set; }
+        public string ItemDropPrototypeName { get; private set; } = "FloorTileItemSteel";
```

Comment:
```suggestion
        public string ItemDropPrototypeName { get; private set; } = string.Empty;
```

This gets overridden in LoadFrom regardless so putting that here is misleading.